tests/hardhat/Governance/GovernanceBravo/proposeTest.ts

```diff
@@ -0,0 +1,323 @@
+import { BigNumber, Signer } from "ethers";
+import { ethers, network } from "hardhat";
+import { loadFixture, mine } from "@nomicfoundation/hardhat-network-helpers";
+import { smock, MockContract, FakeContract } from "@defi-wonderland/smock";
+import chai from "chai";
+const { expect } = chai;
+chai.use(smock.matchers);
+
+const { encodeParameters } = require("../../../Utils/BSC");
+
+import {
+  GovernorBravoDelegate,
+  GovernorBravoDelegate__factory,
+  XVSVault,
+  XVSStore,
+  XVS,
+} from "../../../../typechain";
+import { ProposalType } from "../../util/Proposals";
+import { convertToUnit } from "../../../../helpers/utils";;
+
+let root: Signer;
+let customer: Signer;
+let accounts: Signer[];
+let governorBravoDelegate: MockContract<GovernorBravoDelegate>;
+let xvsVault: FakeContract<XVSVault>;
+let xvsStore: FakeContract<XVSStore>;
+let xvsToken: FakeContract<XVS>;
+
+type GovernorBravoDelegateFixture = {
+  governorBravoDelegate: MockContract<GovernorBravoDelegate>;
+  xvsVault: FakeContract<XVSVault>;
+  xvsStore: FakeContract<XVSStore>;
+  xvsToken: FakeContract<XVS>;
+};
+
+async function governorBravoFixture(): Promise<GovernorBravoDelegateFixture> {
```

Comment:
It looks like this function is used in every test. I can image that future tests might also share fixtures. Does it make sense to add a fixture director at `tests/hardhat/fixtures`?

Reply:
yeah this would be a possible thing when we get back to optimizing the tests